### ISSUE TITLE
iron-image accessibility report 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-image#6]
Here are the audit results: 

[Severe] Elements with ARIA roles must use a valid, non-abstract ARIA role (20)[Warning] Images should have an alt attribute (6)[Warning] Text elements should have a reasonable contrast ratio (1)[Warning] These elements are focusable but either invisible or obscured by another element (9)Not applicable tests (9)Passing tests (4)

Issues found: 

The text that says "loading" over the image box is too light. We need to improve that contrast. 

In terns of proper labels, all of the images just say "image". I knnow this will be up to the developer to set though. 

Also, when I click on the button to load or reload, I do not hear an announcement that the image is loading, or that the load is complete. We should consider how to convey this status. 

With ChromeVox / the keyboard on Chrome OS, I am not able to access the images -- only the written text and buttons. I am able to access them with VoiceOver on Mac though, and hear the (lack of) alt text.

**Comment by alice** [comment]
NB: the role issue is my fault: the audit doesn't recognise role=none currently, but it is valid.

**Comment by alice** [comment]
API accessibility issues:
Needs an alt attribute analogous to `img`.
What is the purpose of `<content>` in this element?

**Comment by lpalmaro** [comment]
Just tested again today. This generally seems good now. Please note that there is no alt ext for the images. They just speak "image". We'll want to help encourage developers to use alt text for images here.
